frame/benchmarking/src/utils.rs

```diff
@@ -321,17 +321,17 @@ pub trait BenchmarkingSetup<T, I = ()> {
 }
 
 /// Grab an account, seeded by a name and index.
-pub fn account<AccountId: Decode + Default>(
+pub fn account<AccountId: Decode>(
 	name: &'static str,
 	index: u32,
 	seed: u32,
 ) -> AccountId {
 	let entropy = (name, index, seed).using_encoded(blake2_256);
-	AccountId::decode(&mut &entropy[..]).unwrap_or_default()
+	AccountId::decode(&mut &entropy[..]).expect("`AccountId` type must be able to decode from any 32 bytes; qed")
```

Comment:
This expectation could surface the places where we create a zero account, so :+1: